recipes/pyacvd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyacvd" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: pyacvd
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyacvd-{{ version }}.tar.gz
+  sha256: acd4579082686d81244896848aaf0b59f9929d85e5bc98bae70bf60ae783bf9a
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py==34]
```

Comment:
Might as well eliminate the skip:
```suggestion
```
We only build against >=3.6 unless someone goes to extra effort.